internal/updater/updater_test.go

```diff
@@ -0,0 +1,457 @@
+package updater
+
+import (
+	"context"
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	kscheme "k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+)
+
+var (
+	schemeBuilder = runtime.NewSchemeBuilder(
+		kscheme.AddToScheme,
+		rukpakv1alpha1.AddToScheme,
+	)
+	bundle = &rukpakv1alpha1.Bundle{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "olm-crds",
+		},
+		Spec: rukpakv1alpha1.BundleSpec{
+			ProvisionerClassName: "core.rukpak.io/plain",
```

Comment:
Note: This provisioner ID was renamed in #100 to core.rukpak.io/plain-v0.

Reply:
Thanks.  I'll update.